doc/docs/getting_started/multi_node.md

```diff
@@ -76,10 +76,10 @@ Once the services are up and running, the journald logs on the **laptop** should
 ```bash
 $ journalctl -u bluechi-controller
 
-Sep 14 14:51:58 laptop systemd[1]: Started BlueChi systemd service controller manager daemon.
-Sep 01 14:51:58 laptop bluechi[3750775]: 2023-09-14 14:51:58,685+0200 INFO        ../src/manager/manager.c:924 manager_start        msg="Starting bluechi 0.5.0"
-Sep 01 14:51:58 laptop bluechi[3750775]: 2023-09-14 14:51:58,928+0200 INFO        ../src/manager/node.c:870 node_method_register    msg="Registered managed node from fd 8 as 'laptop'"
-Sep 01 14:52:02 laptop bluechi[3750775]: 2023-09-14 14:52:02,534+0200 INFO        ../src/manager/node.c:870 node_method_register    msg="Registered managed node from fd 9 as 'pi'"
+Sep 14 14:51:58 laptop systemd[1]: Started BlueChi systemd service controller controller daemon.
+Sep 01 14:51:58 laptop bluechi[3750775]: 2023-09-14 14:51:58,685+0200 INFO        ../src/controller/controller.c:924 controller_start        msg="Starting bluechi 0.5.0"
```

Comment:
I'd also change the version from 0.5.0 tp 0.7.0, this is the version where we are renaming things

Reply:
I am not sure if we should update all the versions here now. It would probably be good to check if these examples here work still as in previous version. How about we create a dedicated issue for that? @mwperina

Reply:
That's more work than just changing the version, but OK

Reply:
Just saw that the logs have been changed anyway, e.g. `../src/manager/manager.c` to `../src/controller/controller`. In that case you are right, lets just change the version. 
I just wanted to make sure that the output here in the doc reflects the actual output.

Reply:
The logs are not the same, and they need to be updated.
On version updating... I have updated them, but should we update them with every release?

Reply:
No, I think it's worth changing the version only in this case, because of the big rename

Reply:
Yes. It'd be great if we'd have versioned documentation (also regarding new features, etc.), but that is something for the future :)